geniza/corpus/tests/test_corpus_admin.py

```diff
@@ -254,6 +254,10 @@ def test_tabulate_queryset(self):
                 f"https://example.com/admin/corpus/document/{doc.id}/change/"
                 in doc_data
             )
+            # initial input should be before last modified
+            # (document fixture has a log entry, so should have a first input)
+            if doc_data[-6]:
+                assert doc_data[-6] < doc_data[-5]
```

Comment:
could throw a message on here for when the assert fails so that we know the pgpid, just in case